src/main/webapp/index.html

```diff
@@ -3,14 +3,15 @@
   <head>
       <meta charset="utf-8" />
       <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
-      <title>noms</title>
+      <title>find events</title>
```

Comment:
Isn't this the home page? Shouldn't this title be on the find-events.html page?

Reply:
Whoops, sorry! I mixed up the two pages